sgbackend/mail.py

```diff
@@ -104,8 +104,28 @@ def _build_sg_mail(self, email):
                 for k, v in email.substitutions.items():
                     personalization.add_substitution(Substitution(k, v))
 
+        # SendGrid does not support adding Reply-To as an extra
+        # header, so it needs to be manually removed if it exists.
+        reply_to_string = ""
         for k, v in email.extra_headers.items():
```

Comment:
please rename to `key`, `value`

Reply:
the entire file uses the `k, v` naming, so it makes sense to keep the consistency.